Makefile

```diff
@@ -438,6 +439,9 @@ create-management-cluster: $(KUSTOMIZE) $(ENVSUBST)
 	sleep 10
 	@echo 'Set kubectl context to the kind management cluster by running "kubectl config set-context kind-capz"'
 
+	# aad-pod-identity deploylment for handling identities 
+	$(ENVSUBST) < $(TEMPLATES_DIR)/addons/aad-pod-identity-deployment.yaml | kubectl apply -f -
```

Comment:
how will this work for other ways of creating Azure clusters (Tilt, clusterctl, kustomize, etc.)? should we consider adding it to infrastructure-components?

Reply:
Currently in the process of getting things to work, was going to add it to the tiltfile, didn’t think about infrastructure components